docs/gameserverapi/README.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: GameServer API
+nav_order: 9
+has_children: true
+---
+
+# GameServer API
+
+The GameServer API is a RESTful API service that facilitates access to Thundernetes' Custom Resources: GameServerBuilds, GameServer, and GameServerDetails. It is an alternative for people who don't want to use tools like [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/), and an easy way to integrate Thundernetes to your own applications. It also allows you to use our [Thundernetes UI](../thundernetesui/README.md).
+
+## How to install the GameServer API
```

Comment:
should we add the link on how to secure the GameServer API?

Reply:
I think I do below, in how to install

Reply:
oh sorry, it was in gray colors because of the "_note" so I skipped it.